static.config.js

```diff
@@ -894,19 +894,19 @@ export default {
   // basePath // Do not alter this line if you want a working PR
   basePath: process.env.BASE_PATH_PR ? process.env.BASE_PATH_PR : '/',
   stagingSiteRoot: 'https://janis-staging.herokuapp.com/',
-  // maxThreads: 2,
+  maxThreads: 2,
```

Comment:
did maxThreads make a difference? its something I added when i was trying to prevent timing out on my own machine. "By default this is set to Infinity to use all available threads on the machine React Static is running on." 

I say remove it unless we know its helping